apps/hpc-cdm/src/app/components/enketo/modals/assignedUsersModal.tsx

```diff
@@ -53,8 +53,8 @@ const AssignedUsersModal = (props: Props) => {
       </UserTitle>
 
       <DialogContent id="alert-dialog-description">
-        {assignment.assignedUsers.map((user) => (
-          <UserLink href={`mailto:${user.email}`}>
+        {assignment.assignedUsers.map((user, i) => (
+          <UserLink key={i} href={`mailto:${user.email}`}>
```

Comment:
We could have user `user.email` as a key here, since it is unique, but since these items are static and won't change, then let's use index as a key, which can be dangerous, see my other comments.